crypto/evp/evp_enc.c

```diff
@@ -1090,7 +1090,7 @@ int EVP_CIPHER_CTX_ctrl(EVP_CIPHER_CTX *ctx, int type, int arg, void *ptr)
     case EVP_CTRL_SET_PIPELINE_OUTPUT_BUFS: /* Used by DASYNC */
     case EVP_CTRL_INIT: /* TODO(3.0) Purely legacy, no provider counterpart */
     default:
-        return EVP_CTRL_RET_UNSUPPORTED;
+        goto conclude;
```

Comment:
Is this supposed to be ret = UNSUPPORTED here??

Reply:
It is already set to that value.

Reply:
Yup.  Look at line 1065

Reply:
While you're here, please fix the inconsistency and rename the label to end or err like everywhere else in openssl.

Reply:
Meh :unamused: ...  but ok